### PR TITLE
[Maistra-1215][Maistra-1211] This reverts the BUILD_TYPE changes

### PR DESCRIPTION
**Pull request by brian-avery**
This is part 2 of https://github.com/Maistra/istio-operator/pull/352

### CODE REVIEW

File: istio-operator/istio-operator.spec [hunk 1]
```diff
@@ -64,16 +64,10 @@ mkdir -p OPERATOR/src/github.com/maistra/istio-operator
...
-  export BUILD_TYPE="maistra"
-else
-  export BUILD_TYPE="servicemesh"
-fi
-
 cd OPERATOR
 export GOPATH=$(pwd):%{gopath}
 pushd src/github.com/maistra/istio-operator/
-VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} make compile collect-resources
+COMMUNITY=%{community_build} GO111MODULE=on VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} make compile collect-resources collect-olm-manifests
```
Comment (rcernich) [inline istio-operator/istio-operator.spec]: `collect-resources` task should include (require) `collect-olm-manifests`.

Comment (brian-avery) [reply istio-operator/istio-operator.spec]: I thought about that, but `collect-resources` seems to refer to the resources directory which this is not part of so I kept them separate.

**Review by rcernich — APPROVED** [review]

**Review by dgn — APPROVED** [review]

**Comment by brian-avery** [comment]
Thanks. Added a WIP label so that I can update the SHA once the operator changes are approved.

**Comment by brian-avery** [comment]
@mergifyio refresh

**Comment by brian-avery** [comment]
@Mergifyio refresh

**Comment by brian-avery** [comment]
@Mergifyio refresh

**Pull request merged**